packages/riipen-ui/src/components/ListItem.jsx

```diff
@@ -10,30 +12,80 @@ class ListItem extends React.Component {
     children: PropTypes.any,
 
     /**
-     * Array of additional CSS classes to use.
+     * Array or string of additional CSS classes to use.
      *
-     * @type {Array}
+     * @type {string | Array}
+     */
+    classes: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
+
+    /**
+     * Whether the list item can be selected
+     */
+    disabled: PropTypes.bool,
+
+    /**
+     * Whether the list item is currently selected
+     */
+    selected: PropTypes.bool,
+
+    /**
+     * Callback for when the list item is clicked
+     */
+    onClick: PropTypes.func,
+
+    /**
+     * The color of the component. It supports those theme colors that make sense for this component.
      */
-    classes: PropTypes.array
+    color: PropTypes.oneOf(["primary", "secondary"])
   };
 
   static defaultProps = {
-    classes: []
+    color: "primary"
   };
 
+  static contextType = ThemeContext;
+
   render() {
-    const { children, classes } = this.props;
+    const {
+      children,
+      classes,
+      onClick,
+      disabled,
+      selected,
+      color
+    } = this.props;
+
+    const className = clsx(classes, color, "list-item", {
+      disabled,
+      selected
+    });
+
+    const theme = this.context;
 
-    const className = clsx(classes);
+    const handleClick = event => {
+      if (onClick && !disabled) onClick(event);
+    };
 
     return (
       <React.Fragment>
-        <li className={className}>{children}</li>
+        <div onClick={handleClick} className={className}>
+          {children}
+        </div>
         <style jsx>{`
-          li {
-            list-style-type: none;
+          .list-item {
+            padding: ${theme.spacing(2)}px ${theme.spacing(3)}px;
+            color: ${theme.palette.text.secondary};
+            font-family: ${theme.typography.body1.fontFamily};
+            font-size: ${theme.typography.body1.fontSize};
+            font-weight: ${theme.typography.body1.fontWeight};
+            line-height: ${theme.typography.body1.lineHeight};
+            letter-spacing: ${theme.typography.body1.letterSpacing};
+            outline: 0;
+            border: none;
+            text-align: inherit;
+            border-left: 5px solid transparent;
             margin: 0;
-            padding: 8px 0;
+            box-sizing: border-box;
```

Comment:
Can we put these in alphabetical order? We usually do that with all our CSS on all projects, but we haven't been able to figure out how to add a styled-jsx linter to this project yet.